src/resolvers.ts

```diff
@@ -11,17 +11,30 @@ export interface UserInput {
   birthdate: string;
 }
 
+function login(parent: any, args: { input: { email: string; password: string } }) {
+  const loggedUser = new User();
+
+  Object.assign(loggedUser, {
+    id: 12,
+    name: 'Rodrigo',
+    email: 'rodrigo@email.com',
+    birthdate: '01-01-1980',
+  });
+
+  return { user: loggedUser, token: 'the_token' };
+}
```

Comment:
Acho que já deu pra perceber tbm que esse arquivo `resolvers` vai começar a crescer batante né? Chegou a refletir sobre isso tbm?

Reply:
Realmente não refleti, talvez seria melhor colocar cada function em um arquivo separado.

Reply:
IMO 3 arquivos pequenos é melhor do que 1 grande. Só cuidado com os conflitos com os outros PRs. Se vc for modificar isso, sugiro fazer num PR separado depois de todos esses outros.